File: CMS/src/app/shared/CompanyScope.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CompanyScope } from 'src/app/shared/CompanyScope.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CompanyScopeService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CompanyScope";
    }
}

File: Service/Implement/CompanyScopeService.cs
namespace Service.Implement
{
    public class CompanyScopeService : BaseService<CompanyScope, ICompanyScopeRepository>
    , ICompanyScopeService
    {
    private readonly ICompanyScopeRepository _CompanyScopeRepository;
    public CompanyScopeService(ICompanyScopeRepository CompanyScopeRepository) : base(CompanyScopeRepository)
    {
    _CompanyScopeRepository = CompanyScopeRepository;
    }
    }
    }

